test/arm/neon/cmla_rot180_lane.c

```diff
@@ -0,0 +1,1026 @@
+        SIMDE_FLOAT32_C(586838.562500), SIMDE_FLOAT32_C(587489.312500)}},
+      {{SIMDE_FLOAT32_C(998.31), SIMDE_FLOAT32_C(538.40),
+        SIMDE_FLOAT32_C(-191.12), SIMDE_FLOAT32_C(-434.48)},
+       {SIMDE_FLOAT32_C(592.83), SIMDE_FLOAT32_C(820.32),
+        SIMDE_FLOAT32_C(-296.84), SIMDE_FLOAT32_C(-612.30)},
+       {SIMDE_FLOAT32_C(-552.34), SIMDE_FLOAT32_C(329.08),
+        SIMDE_FLOAT32_C(765.26), SIMDE_FLOAT32_C(-531.08)},
+       INT32_C(0),
+       {SIMDE_FLOAT32_C(328442.062500), SIMDE_FLOAT32_C(327982.156250),
+        SIMDE_FLOAT32_C(-164147.734375), SIMDE_FLOAT32_C(-164391.093750)}},
+      {{SIMDE_FLOAT32_C(52.61), SIMDE_FLOAT32_C(606.93),
+        SIMDE_FLOAT32_C(-894.22), SIMDE_FLOAT32_C(-854.38)},
+       {SIMDE_FLOAT32_C(972.80), SIMDE_FLOAT32_C(-807.39),
+        SIMDE_FLOAT32_C(668.59), SIMDE_FLOAT32_C(-228.19)},
+       {SIMDE_FLOAT32_C(-528.51), SIMDE_FLOAT32_C(730.93),
+        SIMDE_FLOAT32_C(-230.95), SIMDE_FLOAT32_C(-140.17)},
+       INT32_C(1),
+       {SIMDE_FLOAT32_C(-710996.062500), SIMDE_FLOAT32_C(-710441.750000),
+        SIMDE_FLOAT32_C(-489586.718750), SIMDE_FLOAT32_C(-489546.875000)}},
+      {{SIMDE_FLOAT32_C(556.73), SIMDE_FLOAT32_C(-701.90),
+        SIMDE_FLOAT32_C(-356.50), SIMDE_FLOAT32_C(-532.42)},
+       {SIMDE_FLOAT32_C(856.94), SIMDE_FLOAT32_C(-261.67),
+        SIMDE_FLOAT32_C(-208.07), SIMDE_FLOAT32_C(27.93)},
+       {SIMDE_FLOAT32_C(924.32), SIMDE_FLOAT32_C(-863.60),
+        SIMDE_FLOAT32_C(-687.65), SIMDE_FLOAT32_C(238.39)},
+       INT32_C(0),
+       {SIMDE_FLOAT32_C(-791530.062500), SIMDE_FLOAT32_C(-792788.687500),
+        SIMDE_FLOAT32_C(191966.765625), SIMDE_FLOAT32_C(191790.843750)}},
+      {{SIMDE_FLOAT32_C(-286.79), SIMDE_FLOAT32_C(630.61),
+        SIMDE_FLOAT32_C(-989.22), SIMDE_FLOAT32_C(223.21)},
+       {SIMDE_FLOAT32_C(812.31), SIMDE_FLOAT32_C(667.33),
+        SIMDE_FLOAT32_C(841.41), SIMDE_FLOAT32_C(735.52)},
+       {SIMDE_FLOAT32_C(308.52), SIMDE_FLOAT32_C(-189.06),
+        SIMDE_FLOAT32_C(-63.33), SIMDE_FLOAT32_C(837.76)},
+       INT32_C(1),
+       {SIMDE_FLOAT32_C(153288.531250), SIMDE_FLOAT32_C(154205.937500),
+        SIMDE_FLOAT32_C(158087.750000), SIMDE_FLOAT32_C(159300.171875)}}};
+
+  for (size_t i = 0; i < (sizeof(test_vec) / sizeof(test_vec[0])); i++) {
+    simde_float32x4_t r_ = simde_vld1q_f32(test_vec[i].r_);
+    simde_float32x4_t a = simde_vld1q_f32(test_vec[i].a);
+    simde_float32x4_t b = simde_vld1q_f32(test_vec[i].b);
+    simde_float32x4_t r;
+    SIMDE_CONSTIFY_2_(
+        simde_vcmlaq_rot180_laneq_f32, r,
+        (HEDLEY_UNREACHABLE(), simde_vdupq_n_f32(SIMDE_FLOAT32_C(0.0))),
+        test_vec[i].lane, r_, a, b);
+    simde_test_arm_neon_assert_equal_f32x4(r, simde_vld1q_f32(test_vec[i].r),
+                                           1);
+
+    // simde_float32x4_t r = simde_vcmlaq_rot180_laneq_f32(r_, a, b,
+    // test_vec[i].lane); simde_test_arm_neon_write_f32x4(2, r,
+    // SIMDE_TEST_VEC_POS_LAST);
```

Comment:
Please tidy up your test code as well; thank you!
```suggestion
```

Reply:
Hello! I've cleaned up the code and reformatted it using Clang-Format with the LLVM style. Apologies for the coding style and redundant comments.